ckanext/datavic_iar_theme/templates/snippets/dataset_search_form.html

```diff
@@ -10,6 +10,21 @@
 
     <div class="search-form-wrapper">
       <div class="rpl-search-form da-search rpl-site-constrain--on-all">
+        {% if query %}
+          <h1> {{ _('Search Results') }} ({{count}} 
```

Comment:
It can be written as: 
```html
<h1>
   {{ ungettext(
          'Search Results({number} dataset found)', 
          'Search Results({number} datasets found)', 
          count
    ).format(number=count) }}
</h1>
```

`ungettext` is a helper that accepts singular and plural forms, and an arbitrary number as a third parameter. It outputs one of the forms, depending on the number's value. Generally, it's a recommended way to handle our case instead of using conditions